src/main/java/it/gov/pagopa/payment/dto/qrcode/AuthPaymentDTO.java

```diff
@@ -0,0 +1,23 @@
+package it.gov.pagopa.payment.dto.qrcode;
```

Comment:
Probably we will use the same model also for the CIE
```suggestion
package it.gov.pagopa.payment.dto;
```